### DISCUSSION TITLE
derived type variable error

### DISCUSSION THREAD

**Discussion by lachlanbelcher** [nwchemgit/nwchem#773] [Q&A]
Hi again,  I'm getting the following error in my builds here:
![image](https://user-images.githubusercontent.com/70237894/234577486-f6ce9b58-30b3-4eec-b5cd-16f85cf8dd3e.png)
However, in the file in question, the type does indeed have the SEQUENCE type:
![image](https://user-images.githubusercontent.com/70237894/234577748-faeb6eab-dcff-452c-b846-be4c1afe5cc5.png)
Can anyone help me understand why the file causes a problem, but only in some builds?  Thanks!

Link to actions:
https://github.com/lachlanbelcher/nwchem/actions/runs/4808126980

**Comment by dmejiar** [comment]
It seems that you have two `sif_data.h` files. The one with the sequence attribute [source/colib/sif_data.h](https://gitlab.com/columbus-program-system/columbus/-/blob/master/Columbus/source/colib/sif_data.h) seems to be overwritten sometimes with the one without that attribute [source/colib/newsif/sif_data.h](https://gitlab.com/columbus-program-system/columbus/-/blob/master/Columbus/source/colib/newsif/sif_data.h)  when this [command](https://github.com/lachlanbelcher/nwchem/blob/6cff426c8f8524aa5a9539bf2fc8f44217a37b41/src/NWints/sifs/build_colib.sh#L30) gets executed.

The nondeterministic behavior might be associated with the concurrent execution of `build_colib.sh` in parallel builds. I suggest two changes: 1) copying only the `sif_data.h` with the sequence attribute, and 2) splitting the GNUMakefile to something like

```make
izero.F:
	./build_colib.sh

blaswrapper.f cdcopy.f\
colib1.f colib2.f colib3.f colib4.f \
colib5.f colib6.f colib7.f colib8.f \
colib8a.f colib9.f: izero.F
```

I think you would also be missing `bummer` , right?

**Comment by lachlanbelcher** [comment]
Thank you, that is a good catch.  I made some changes and will test again.  Bummer is now in colib7.f, so I need to remove that instruction.
